apps/omg_watcher/test/test_helper.exs

```diff
@@ -12,9 +12,11 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-ExUnit.configure(exclude: [integration: true])
+ExUnit.configure(exclude: [integration: true, property: true, wrappers: true])
 ExUnitFixtures.start()
 ExUnit.start()
 
 {:ok, _} = Application.ensure_all_started(:httpoison)
 {:ok, _} = Application.ensure_all_started(:fake_server)
+
+Mix.Task.run("ecto.migrate", ~w(--quiet))
```

Comment:
why was this added now and wasn't needed beforehand?

Reply:
bad as it may sound, I have not the faintest idea. Only thing I know is that I have seen other people including such migrations in their `test_helpers.exs` or in their manual processes, when I searched for the issue.

The issue was that sometimes (that is, unless one has done `MIX_ENV=test ecto.migrate` manually), tests would complain about things like tables not being there in the db. 

At first I thought that it makes sense for it to be there and I can just yolo the fact that it worked without before. On 2nd thought this deserves proper investigation...

Reply:
I suspect I didn't work after #562, but nobody really drop tables from test database... but now nobody could be sure ;)

Reply:
that's entirely possible, but how would these migrated test dbs emerge in the 1st place?

Reply:
hm, I fiddled around with this, and it seems (locally, at least) this https://github.com/omisego/elixir-omg/pull/828/commits/a513ec255e9d1e9995834108a464f1ce12e0740f is also a viable alternative.

Weirdly enough, now, if I don't apply neither of these fixes, it complains about sth different:

```
     ** (EXIT from #PID<0.4439.0>) shutdown: failed to start child: OMG.Watcher.DB.Repo
         ** (EXIT) shutdown: failed to start child: DBConnection.Ownership.Manager
             ** (EXIT) an exception was raised:
                 ** (MatchError) no match of right hand side value: {:error, {:noproc, {GenServer, :call, [DBConnection.Watcher, {:watch, DBConnection.ConnectionPool.Supervisor, {DBConnection.ConnectionPool.Pool, {#PID<0.4457.0>, #Reference<0.1063356787.2548695042.105969>, Postgrex.Protocol, [types: Postgrex.DefaultTypes, port: 5432, repo: OMG.Watcher.DB.Repo, telemetry_prefix: [:omg, :watcher, :db, :repo], otp_app: :omg_watcher, pool_size: 10, adapter: Ecto.Adapters.Postgres, timeout: 60000, connect_timeout: 60000, ownership_timeout: 180000, username: "omisego_dev", password: "omisego_dev", database: "omisego_test", hostname: "localhost"]}}}, :infinity]}}}
                     lib/db_connection/ownership/manager.ex:60: DBConnection.Ownership.Manager.init/1
                     (stdlib) gen_server.erl:374: :gen_server.init_it/2
                     (stdlib) gen_server.erl:342: :gen_server.init_it/6
                     (stdlib) proc_lib.erl:249: :proc_lib.init_p_do_apply/3
```

I'll poke around some more and come up with the least frightening option to distill a reviewable PR finally.

Reply:
:man_facepalming: okay, so the `migrate` has always been there https://github.com/omisego/elixir-omg/blob/master/apps/omg_watcher/mix.exs#L56. It just isn't executed when doing the modified `coveralls --umbrella` task. Brawo ja.